src/poliastro/constants/general.py

```diff
@@ -489,7 +489,7 @@
 R_polar_jupiter = Constant(
     "R_polar_jupiter",
     "Jupiter polar radius",
-    6.6854,
+    6.6854e7,
```

Comment:
This is because the branch is not up to date with current `master`. But let's not be picky with that now.